neural_lam/vis.py

```diff
@@ -179,3 +181,90 @@ def plot_spatial_error(error, title=None, vrange=None):
         fig.suptitle(title, size=10)
 
     return fig
+
+@matplotlib.rc_context(utils.fractional_plot_bundle(1))
+def verify_inference(file_path:str, feature_channel:int, vrange=None, save_path=None):
+    """
+    Plot example prediction, forecast, and ground truth.
+    Each has shape (N_grid,)
+    """
+
+    # Verify that feature channel is between 0 and 42
+    if not 0 <= feature_channel <= 42:
```

Comment:
the number of feature channels is defined in constants.py by the user. 42 is only the current setting. This if clause should probably compare to the size of the last dimension of the .npy array from file_path. Or alternatively calculate them from `contants.py` param_names and is vertical lists.

Reply:
fixed in 7715d2be8001de4a2853cf448c8699efb29dee23